docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -54,10 +54,12 @@ The top bar of Teleport Connect consists of:
   Teleport clusters as well as log in or out of the clusters.
 - The **connection list** (the top left) showing recent connections, allowing you to seamlessly switch
   between them.
-- The **command bar** (in the middle), which launches any command you input in a new terminal tab.
-- The **cluster selector** (to the left of the command bar), which shows up only if you have set up
+- The **search bar** (in the middle), which allows you to search for resources across clusters.
+- The **cluster selector** (to the left of the search bar), which shows up only if you have set up
   Trusted Clusters and there are leaf clusters connected to the root cluster. It lets you browse
-  leaf cluster resources. It also changes which cluster the commands in the command bar are going to target.
+  leaf cluster resources. Also, the "Open new terminal" action will bound new terminal tabs to the selected cluster.
+- The **additional actions menu** (to the left of the profile selector), containing options such as
+  opening a config file or creating an access request (if you are connected to an Enterprise cluster).
```

Comment:
I'm not sure about the access request part, what bothers me is that additional explanation in parentheses. We're in the section which explains how the UI looks and that conditional in the parens puts too much focus on this additional context of enterprise clusters. It's a little bit too wordy in my opinion.

OTOH there's really no other action we could mention as "Open new terminal" is mentioned above and the other two actions are macOS only.

What do you think about this then?

```suggestion
  opening a config file or creating an access request in an Enterprise cluster.
```

Reply:
Thanks for this small improvement, it indeed reads better 👍